packages/tools/devtools/devtools-core/api-report/devtools-core.api.md

```diff
@@ -89,7 +89,7 @@ export namespace ContainerDevtoolsFeatures {
     }
 }
 
-// @beta
+// @alpha
```

Comment:
We don't strictly need to make these `devtools-core` types beta yet, and they reference core framework concepts that are currently `@alpha`. This changes makes them `@alpha` to match.

Reply:
Technically breaking, but we have not released this package publicly yet.